webapp/app/js/directives/clonejobs.js

```diff
@@ -93,6 +93,96 @@ treeherder.directive('thCloneJobs', [
     });
 
     $rootScope.$on(
+        thEvents.selectNextJob, function(ev){
+
+        var jobMap = ThResultSetModel.getJobMap($rootScope.repoName);
+        var lastJobSelected = ThResultSetModel.getSelectedJob($rootScope.repoName);
+        var el, key, job, step;
+        var step = 'nextElementSibling'
+        
+        if(lastJobSelected.el == undefined || lastJobSelected.el[0] == undefined ){
+
+            var currentElement = document.getElementsByClassName('result-set')[0];
+            el = traceJob($(currentElement),step,'job-btn');
+
+        }else{
+
+            var currentElement = lastJobSelected.el[0];    
+
+            //Parsing for parent which satisy parameter mentioned in variable step
```

Comment:
I'm glad you are commenting as much as possible! There is no such thing as over-commented code. A minor typo here, I assume you meant *satisfy*?

Also general style for js comments is single character of whitespace after the `//`. eg.
`// My comment`